### PR TITLE
test-x64.py: Support parallel testing on x86 and x64

### PR DESCRIPTION
**Pull request by legendlee1314**
1. Flashing phase can be run parallelly.
2. Chromium will be sync at one time in order to use uniform revision.
3. Enable parallel testing only now.

### CODE REVIEW

**Comment by legendlee1314** [comment]
@gyagp , please help review.

**Comment by legendlee1314** [comment]
@gyagp , please review again.

File: python/test-x64.py [hunk 1]
```diff
@@ -112,24 +114,39 @@ def test():
...
+            error(dir_aosp + ' does not exist')
+        backup_dir(dir_aosp)
+
+        pool = Pool(processes=len(target_archs))
+        for arch in target_archs:
             cmd = python_aosp + ' --target-arch %s --target-type %s --device-id %s --flash-image ' % (arch, args.target_type, devices_info[arch])
             cmd = suffix_cmd(cmd, args, log)
-            execute(cmd, abort=True, interactive=True, dryrun=dryrun)
-            restore_dir()
+            pool.apply_async(execute, args=(cmd,), kwds=dict(abort=True, interactive=True, dryrun=True))
```
Comment (gyagp) [inline python/test-x64.py]: dryrun=dryrun here

File: python/test-x64.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 
+from multiprocessing import Pool
```
Comment (gyagp) [inline python/test-x64.py]: Remove this import

**Comment by legendlee1314** [comment]
@gyagp , done

**Pull request merged**